data.sql

```diff
@@ -1,5 +1,44 @@
 /* Populate database with sample data. */
+INSERT INTO
+  animals(
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES
+  ('Agumon', '2022-02-03', 0, true, 10.23);
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO
+  animals(
+    name,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES
+  ('Gabumon', '2018-15-11', 2, true, 8);
```

Comment:
- [ ] Kindly insert data here with a good date format. I am getting an error when inserting this line because your date format is using the format `yy-day-month` here. Kindly correct it to `yy-mm-day` => '2018-11-15`

![image](https://user-images.githubusercontent.com/65068771/159371643-c7bfbcd0-c7b3-4dc8-967e-bedd0e26f92b.png)

Reply:
Hi @andersonlebon, firstly thank you for taking out the time to review my project. I first tried it the way you suggested here but I was getting the same error as you are "Perhaps you need a different datestyle' setting". I think the datestyle for both our PC's have different default settings and if I changed it, maybe a 3rd person who has dd-mm-yy  will get the same error. But if you really recommend it, i'd be more than happy to change.

Reply:
hey @zhadier  the default format of date in posgres is `yy-mm-day` [see here](https://www.geeksforgeeks.org/postgresql-date-data-type/#:~:text=PostgreSQL%20uses%20the%20yyyy%2Dmm,CURRENT_DATE%20after%20the%20DEFAULT%20keyword.)

check this :point_down: 

![image](https://user-images.githubusercontent.com/65068771/159374229-090d496d-9e69-4552-857c-11399ad2f40a.png)